model/src/main/kotlin/FileFormat.kt

```diff
@@ -83,6 +84,23 @@ fun File.mapper() = FileFormat.forFile(this).mapper
  */
 inline fun <reified T : Any> File.readValue(): T = mapper().readValue(this)
 
+/**
+ * Use the Jackson mapper returned from [File.mapper] to read an object of type [T] from this file, or return null if
+ * the file has no content.
+ */
+inline fun <reified T : Any> File.readValueOrNull(): T? =
+    mapper().readTree(this)?.let {
+        // Parse the file in a two-step process to avoid readValue() throwing an exception on empty files. Also see
+        // https://github.com/FasterXML/jackson-databind/issues/1406#issuecomment-252676674.
+        mapper().treeToValue(it)
+    }
```

Comment:
Have you considered the impact on memory consumption for large files (ORT results)?

Reply:
No, because 1) it's up to the caller to decide which of the `readValue` functions to use, and 2) I deliberately start using it only for (rather small) configuration files (as mentioned in the commit message), not for ORT result files.